rythmbadge/sampling_lib.c

```diff
@@ -104,7 +105,10 @@ ret_code_t sampling_start_microphone(nrf_pdm_mode_t mode)
 	if (audio_switch_get_position()==OFF)
 		return ret;
 
-	ret = drv_audio_init(mode);
+	memset(timestamp_buffer, 0, sizeof(timestamp_buffer));
+	uint64_t initial_recording_timestamp = systick_get_millis();
```

Comment:
I've tested this commit together with the script to decode the timestamps from #29 and I'm getting years going from 1970 to 2077 and hours that don't make sense either. Is this an error in #29 or is there a problem with how the timestamps are recorded here?

Reply:
#29 seems outdated, it's still assuming the initial metadata file format that was modified furing the review of this PR

Reply:
Cool, thanks for the clarification, then I'm marking again as good to merge :)